src/SourceBuild/content/eng/pipelines/templates/steps/publish-test-results-pr.yml

```diff
@@ -0,0 +1,37 @@
+parameters:
+- name: pipelineMode
```

Comment:
This parameter could benefit from some documentation as it is not obvious to me the purpose.